lib/oli_web/templates/project/overview.html.eex

```diff
@@ -59,15 +61,13 @@
 
         </div>
       <% end %>
+    </div>
 
-    <%= submit "Save", class: "float-left btn btn-md btn-primary mt-2" %>
-      <div class="d-flex flex-row">
-        <div class="flex-grow-1"></div>
-        <%= submit "Save", class: "float-left btn btn-md btn-primary mt-2" %>
-      </div>
+    <div>
+      <%= submit "Save", class: "btn btn-md btn-primary mt-2" %>
```

Comment:
It looks like there are still 2 save buttons that both belong to the same form. Is this intentional?